ibtk/include/ibtk/FEDataManager.h

```diff
@@ -220,6 +232,26 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
         bool use_nodal_quadrature;
     };
 
+    /*!
+     * \brief Struct WorkloadSpec encapsulates the parameters used to
+     * calculate workload estimates (i.e., the input to the load balancing
+     * algorithm).
+     */
+    struct WorkloadSpec
+    {
+        /// Whether or not to clear the workload index arrays at the start of
+        /// the calculation. This should be true unless some other object is
+        /// also adding values into the workload estimate.
+        bool clear_estimate = true;
+
+        /// The multiplier applied to each quadrature point.
+        double q_point_weight = 2.0;
+
+        /// Multiplier on Eulerian cells. At the present time this is always
+        /// assumed to be 1.
+        static constexpr double eulerian_cell_weight = 1.0;
```

Comment:
Do we need this parameter? I am understanding this to say: `work_estimate(i,j,k) = eulerian_cell_weight + n_q_points(i,j,k) * q_point_weight`. Because the actual value of the workload estimate does not matter, we can always divide through by eulerian_cell_weight and get: `work_estimate(i,j,k) = 1 + n_q_points(i,j,k) * q_point_weight_prime` with `q_point_weight_prime = q_point_weight / eulerian_cell_weight`.

Reply:
Technically no, and getting ODR worked out for this is complicated. I'll just document that it is always 1.